components/playlist/browser/resources/components/app.v1.tsx

```diff
@@ -37,6 +37,10 @@ const StyledHeader = styled(Header)`
   height: var(--header-height);
 `
 
+const StyledAlertCenter = styled(AlertCenter)`
+  --leo-alert-center-position: relative;
```

Comment:
Looks like this hasn't quite made it in yet sorry @sangwoo108 

Will this break anything if it changes from absolute to relative in the future?

Reply:
Ah, you're right. I should revert this 👍